README.md

```diff
@@ -276,6 +276,29 @@ func HandleRequest(ctx context.Context, name string) (string, error) {
 }
 ```
 
+## Integrations
+
+Support for incoming requests with [valyala/fasthttp](https://github.com/valyala/fasthttp):
+
+```go
+func index(ctx *fasthttp.RequestCtx) {
+	ctx.WriteString("Welcome!")
+}
+
+func hello(ctx *fasthttp.RequestCtx) {
+	fmt.Fprintf(ctx, "Hello, %s!\n", ctx.UserValue("name"))
+}
+
+func main() {
+	fh := xray.NewFastHTTP(nil)
+	r := router.New()
```

Comment:
@nicolascb which go package you're using for `router.New()`?

Reply:
> @nicolascb which go package you're using for `router.New()`?

In this example I am using [github.com/fasthttp/router](https://github.com/fasthttp/router), but it is not mandatory.